hail/src/main/scala/is/hail/expr/ir/MatrixWriter.scala

```diff
@@ -1019,10 +1019,10 @@ case class VCFPartitionWriter(
         _writeB(cb, v.toBytes(cb).loadBytes(cb))
       case v: SCallValue =>
         val ploidy = v.ploidy(cb)
-        cb.if_(ploidy.ceq(0), cb._fatal("VCF spec does not support 0-ploid calls."))
+        cb.if_(ploidy.ceq(0), cb._fatal("VCF does not support 0-ploid calls."))
         cb.if_(
           ploidy.ceq(1) && v.isPhased(cb),
-          cb._fatal("VCF spec does not support phased haploid calls."),
+          cb._fatal("VCFv4.2 does not support phased haploid calls."), // FIXME(chrisvittal): it does as of VCFv4.4 released Jan. 2023, but we
```

Comment:
but we?

Reply:
Thanks for catching this, I rewrote the entire comment